app/views/my/page.html.erb

```diff
@@ -28,10 +28,10 @@ See doc/COPYRIGHT.rdoc for more details.
 ++#%>
 <% breadcrumb_paths(l(:label_my_page)) %>
 <%= toolbar title: l(:label_my_page) do %>
-  <li class="toolbar-item">
+  <li class="toolbar-item" title="<%= l(:label_personalize_page) %>">
     <%= link_to({ action: 'page_layout' }, accesskey: accesskey(:edit), class: 'button') do %>
-      <i class="button--icon icon-edit"></i>
-      <span class="button--text"><%= l(:label_personalize_page) %></span>
+      <i class="button--icon icon-settings"></i>
+      <span class="button--text"></span>
     <% end %>
```

Comment:
As strange as it sounds, the `title` [attribute is not accessible](https://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong/). We should therefore add a `hidden-for-sighted` span.